services/langfuzz/setup.sh

```diff
@@ -80,6 +75,17 @@ retry apt-get install -y -qq --no-install-recommends "${packages[@]}"
 # Generate locales
 locale-gen en_US.utf8
 
+# Add 32 bit packages for x86-64 (unavailable for arm64)
```

Comment:
Sorry, I should have worded it better. I said line 77 because that's where the `apt-get install` was before you removed the i386 & multilib lines from `packages`. You're changing `packages` which is used on (now) line 71.. 🫠

Reply:
Oops.. confused with another script, where they were installed later